diff -uNr e2fsprogs-1.44.2/misc/badblocks.c e2fsprogs-1.44.2.mod/misc/badblocks.c
--- e2fsprogs-1.44.2/misc/badblocks.c	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/misc/badblocks.c	2018-07-08 16:30:46.135596971 +0300
@@ -1007,7 +1007,7 @@
 		fprintf(stderr, _("%s is mounted; "), device_name);
 		if (force) {
 			fputs(_("badblocks forced anyway.  "
-				"Hope /etc/mtab is incorrect.\n"), stderr);
+				"Hope @TERMUX_PREFIX@/etc/mtab is incorrect.\n"), stderr);
 			return;
 		}
 	abort_badblocks:
