diff -uNr e2fsprogs-1.44.2/misc/e4crypt.c e2fsprogs-1.44.2.mod/misc/e4crypt.c
--- e2fsprogs-1.44.2/misc/e4crypt.c	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/misc/e4crypt.c	2018-07-08 16:31:35.392554803 +0300
@@ -613,7 +613,7 @@
 
 static void get_default_salts(void)
 {
-	FILE	*f = setmntent("/etc/mtab", "r");
+	FILE	*f = setmntent("@TERMUX_PREFIX@/etc/mtab", "r");
 	struct mntent *mnt;
 
 	while (f && ((mnt = getmntent(f)) != NULL)) {
