diff -uNr e2fsprogs-1.44.2/doc/Makefile.in e2fsprogs-1.44.2.mod/doc/Makefile.in
--- e2fsprogs-1.44.2/doc/Makefile.in	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/doc/Makefile.in	2018-07-08 16:01:54.974753397 +0300
@@ -17,7 +17,7 @@
 HTML=makeinfo --html --no-split
 PS2PDF=ps2pdf
 
-all:: libext2fs.info libext2fs.dvi libext2fs.html
+all:: libext2fs.info
 
 install-doc-libs: libext2fs.info libext2fs.dvi
 	$(Q) $(RM) -rf $(DESTDIR)$(infodir)/libext2fs.info*
