diff -uNr e2fsprogs-1.44.2/misc/e4defrag.c e2fsprogs-1.44.2.mod/misc/e4defrag.c
--- e2fsprogs-1.44.2/misc/e4defrag.c	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/misc/e4defrag.c	2018-07-08 16:32:15.102790641 +0300
@@ -223,7 +223,7 @@
 
 	fp = setmntent(mtab, "r");
 	if (fp == NULL) {
-		perror("Couldn't access /etc/mtab");
+		perror("Couldn't access @TERMUX_PREFIX@/etc/mtab");
 		return -1;
 	}
 
@@ -290,7 +290,7 @@
 
 	fp = setmntent(mtab, "r");
 	if (fp == NULL) {
-		perror("Couldn't access /etc/mtab");
+		perror("Couldn't access @TERMUX_PREFIX@/etc/mtab");
 		return -1;
 	}
 
