diff -uNr e2fsprogs-1.44.2/misc/ismounted.c e2fsprogs-1.44.2.mod/misc/ismounted.c
--- e2fsprogs-1.44.2/misc/ismounted.c	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/misc/ismounted.c	2018-07-08 16:34:07.623463716 +0300
@@ -197,7 +197,7 @@
 	if (mount_flags)
 		return 1;
 #endif /* __linux__ */
-	retval = check_mntent_file("/etc/mtab", file, &mount_flags);
+	retval = check_mntent_file("@TERMUX_PREFIX@/etc/mtab", file, &mount_flags);
 	if (retval)
 		return 0;
 	return (mount_flags);
