diff -uNr e2fsprogs-1.44.2/misc/e2undo.c e2fsprogs-1.44.2.mod/misc/e2undo.c
--- e2fsprogs-1.44.2/misc/e2undo.c	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/misc/e2undo.c	2018-07-08 16:38:50.495179506 +0300
@@ -240,7 +240,7 @@
 	 */
 	tdb_dir = getenv("E2FSPROGS_UNDO_DIR");
 	if (!tdb_dir)
-		tdb_dir = "/var/lib/e2fsprogs";
+		tdb_dir = "@TERMUX_PREFIX@/var/lib/e2fsprogs";
 
 	if (!strcmp(tdb_dir, "none") || (tdb_dir[0] == 0) ||
 	    access(tdb_dir, W_OK))
