diff -uNr e2fsprogs-1.44.2/util/android_config.h e2fsprogs-1.44.2.mod/util/android_config.h
--- e2fsprogs-1.44.2/util/android_config.h	2018-05-16 02:42:44.000000000 +0300
+++ e2fsprogs-1.44.2.mod/util/android_config.h	2018-07-08 16:34:44.847021077 +0300
@@ -2,7 +2,7 @@
 #define HAVE_MALLOC_H 1
 #endif
 
-#define ROOT_SYSCONFDIR "/etc"
+#define ROOT_SYSCONFDIR "@TERMUX_PREFIX@/etc"
 
 #define ENABLE_LIBSPARSE 1
 
