diff -uNr qemu-2.11.1/hw/i386/multiboot.c qemu-2.11.1.mod/hw/i386/multiboot.c
--- qemu-2.11.1/hw/i386/multiboot.c	2018-02-14 23:53:22.000000000 +0200
+++ qemu-2.11.1.mod/hw/i386/multiboot.c	2018-06-24 01:50:55.075794787 +0300
@@ -192,11 +192,6 @@
         int kernel_size;
         fclose(f);
 
-        if (((struct elf64_hdr*)header)->e_machine == EM_X86_64) {
-            fprintf(stderr, "Cannot load x86-64 image, give a 32bit one.\n");
-            exit(1);
-        }
-
         kernel_size = load_elf(kernel_filename, NULL, NULL, &elf_entry,
                                &elf_low, &elf_high, 0, I386_ELF_MACHINE,
                                0, 0);
