diff -uNr qemu-2.10.1/tcg/tcg.c qemu-2.10.1.mod/tcg/tcg.c
--- qemu-2.10.1/tcg/tcg.c	2017-10-03 00:23:41.000000000 +0300
+++ qemu-2.10.1.mod/tcg/tcg.c	2017-11-29 23:56:00.866992814 +0200
@@ -2999,7 +2999,7 @@
     /* Enable this block to be able to debug the ELF image file creation.
        One can use readelf, objdump, or other inspection utilities.  */
     {
-        FILE *f = fopen("/tmp/qemu.jit", "w+b");
+        FILE *f = fopen("/data/data/com.termux/files/usr/tmp/qemu.jit", "w+b");
         if (f) {
             if (fwrite(img, img_size, 1, f) != img_size) {
                 /* Avoid stupid unused return value warning for fwrite.  */
