diff -uNr qemu-2.11.1/roms/openbios/arch/unix/plugins/loader.c qemu-2.11.1.mod/roms/openbios/arch/unix/plugins/loader.c
--- qemu-2.11.1/roms/openbios/arch/unix/plugins/loader.c	2018-02-14 23:54:42.000000000 +0200
+++ qemu-2.11.1.mod/roms/openbios/arch/unix/plugins/loader.c	2018-06-24 01:50:55.122461456 +0300
@@ -19,7 +19,7 @@
 
 #include "unix/plugins.h"
 
-#define PLUGINDIR  "/usr/share/OpenBIOS/plugins"
+#define PLUGINDIR  "/data/data/com.termux/files/usr/share/OpenBIOS/plugins"
 #define PATHSIZE   256
 
 #define DEBUG_PLUGINS
