diff -uNr qemu-2.11.1/roms/seabios/scripts/kconfig/lkc.h qemu-2.11.1.mod/roms/seabios/scripts/kconfig/lkc.h
--- qemu-2.11.1/roms/seabios/scripts/kconfig/lkc.h	2018-02-14 23:54:43.000000000 +0200
+++ qemu-2.11.1.mod/roms/seabios/scripts/kconfig/lkc.h	2018-06-24 01:50:55.125794790 +0300
@@ -31,7 +31,7 @@
 #define PACKAGE "linux"
 #endif
 
-#define LOCALEDIR "/usr/share/locale"
+#define LOCALEDIR "/data/data/com.termux/files/usr/share/locale"
 
 #define _(text) gettext(text)
 #define N_(text) (text)
