diff -uNr qemu-2.10.1/hw/i386/multiboot.c qemu-2.10.1.mod/hw/i386/multiboot.c
--- qemu-2.10.1/hw/i386/multiboot.c	2017-10-03 00:23:41.000000000 +0300
+++ qemu-2.10.1.mod/hw/i386/multiboot.c	2017-11-29 23:53:53.276992905 +0200
@@ -192,11 +192,6 @@
         int kernel_size;
         fclose(f);
 
-        if (((struct elf64_hdr*)header)->e_machine == EM_X86_64) {
-            fprintf(stderr, "Cannot load x86-64 image, give a 32bit one.\n");
-            exit(1);
-        }
-
         kernel_size = load_elf(kernel_filename, NULL, NULL, &elf_entry,
                                &elf_low, &elf_high, 0, I386_ELF_MACHINE,
                                0, 0);
