diff -uNr qemu-2.10.1/roms/openbios/arch/unix/plugins/loader.c qemu-2.10.1.mod/roms/openbios/arch/unix/plugins/loader.c
--- qemu-2.10.1/roms/openbios/arch/unix/plugins/loader.c	2017-10-03 00:29:31.000000000 +0300
+++ qemu-2.10.1.mod/roms/openbios/arch/unix/plugins/loader.c	2017-11-30 00:00:43.556992612 +0200
@@ -19,7 +19,7 @@
 
 #include "unix/plugins.h"
 
-#define PLUGINDIR  "/usr/share/OpenBIOS/plugins"
+#define PLUGINDIR  "/data/data/com.termux/files/usr/share/OpenBIOS/plugins"
 #define PATHSIZE   256
 
 #define DEBUG_PLUGINS
