diff -uNr minetest-0.4.17.1/src/guiscalingfilter.cpp minetest-0.4.17.1.mod/src/guiscalingfilter.cpp
--- minetest-0.4.17.1/src/guiscalingfilter.cpp	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/guiscalingfilter.cpp	2018-09-04 15:42:26.136821021 +0300
@@ -113,19 +113,6 @@
 			(u32)destrect.getHeight()));
 	imageScaleNNAA(srcimg, srcrect, destimg);
 
-#ifdef __ANDROID__
-	// Android is very picky about textures being powers of 2, so expand
-	// the image dimensions to the next power of 2, if necessary, for
-	// that platform.
-	video::IImage *po2img = driver->createImage(src->getColorFormat(),
-			core::dimension2d<u32>(npot2((u32)destrect.getWidth()),
-			npot2((u32)destrect.getHeight())));
-	po2img->fill(video::SColor(0, 0, 0, 0));
-	destimg->copyTo(po2img);
-	destimg->drop();
-	destimg = po2img;
-#endif
-
 	// Convert the scaled image back into a texture.
 	scaled = driver->addTexture(scalename, destimg, NULL);
 	destimg->drop();
