diff -uNr minetest-0.4.17.1/src/log.cpp minetest-0.4.17.1.mod/src/log.cpp
--- minetest-0.4.17.1/src/log.cpp	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/log.cpp	2018-09-04 15:42:26.140154371 +0300
@@ -114,43 +114,6 @@
 std::ostream infostream(&info_buf);
 std::ostream verbosestream(&verbose_buf);
 
-// Android
-#ifdef __ANDROID__
-
-static unsigned int g_level_to_android[] = {
-	ANDROID_LOG_INFO,     // LL_NONE
-	//ANDROID_LOG_FATAL,
-	ANDROID_LOG_ERROR,    // LL_ERROR
-	ANDROID_LOG_WARN,     // LL_WARNING
-	ANDROID_LOG_WARN,     // LL_ACTION
-	//ANDROID_LOG_INFO,
-	ANDROID_LOG_DEBUG,    // LL_INFO
-	ANDROID_LOG_VERBOSE,  // LL_VERBOSE
-};
-
-class AndroidSystemLogOutput : public ICombinedLogOutput {
-	public:
-		AndroidSystemLogOutput()
-		{
-			g_logger.addOutput(this);
-		}
-		~AndroidSystemLogOutput()
-		{
-			g_logger.removeOutput(this);
-		}
-		void logRaw(LogLevel lev, const std::string &line)
-		{
-			STATIC_ASSERT(ARRLEN(g_level_to_android) == LL_MAX,
-				mismatch_between_android_and_internal_loglevels);
-			__android_log_print(g_level_to_android[lev],
-				PROJECT_NAME_C, "%s", line.c_str());
-		}
-};
-
-AndroidSystemLogOutput g_android_log_output;
-
-#endif
-
 ///////////////////////////////////////////////////////////////////////////////
 
 
