diff -uNr wireshark-2.6.1/ui/gtk/Makefile.am wireshark-2.6.1.mod/ui/gtk/Makefile.am
--- wireshark-2.6.1/ui/gtk/Makefile.am	2018-05-22 21:53:09.000000000 +0300
+++ wireshark-2.6.1.mod/ui/gtk/Makefile.am	2018-06-26 18:49:28.888305122 +0300
@@ -305,8 +305,8 @@
 	$(NODIST_GENERATED_FILES)	\
 	$(BUILD_PIXBUF_CSOURCE)
 
-wireshark-tap-register.c: $(WIRESHARK_TAP_SRC) ../make-taps
-	$(AM_V_GEN)../make-taps $@ $(filter %.c,$^)
+wireshark-tap-register.c: $(WIRESHARK_TAP_SRC) $(top_srcdir)/make-taps.prebuilt
+	$(AM_V_GEN)$(top_srcdir)/make-taps.prebuilt $@ $(filter %.c,$^)
 
 wireshark-gresources.c: main.gresources.xml $(shell glib-compile-resources --sourcedir=$(top_srcdir) --generate-dependencies $(srcdir)/main.gresources.xml)
 	@echo Making $@
