diff -uNr wireshark-2.6.1/epan/dissectors/Makefile.am wireshark-2.6.1.mod/epan/dissectors/Makefile.am
--- wireshark-2.6.1/epan/dissectors/Makefile.am	2018-05-22 21:53:08.000000000 +0300
+++ wireshark-2.6.1.mod/epan/dissectors/Makefile.am	2018-06-26 18:49:28.824971786 +0300
@@ -1967,7 +1967,7 @@
 dissectors.c: make-dissectors $(ALL_DISSECTORS_SRC)
 	@echo Making dissectors.c
 	@echo $^ | tr ' ' '\n' | $(GREP) '\.c$$' > dissectors.in.txt && \
-	    $(builddir)/make-dissectors $@ @dissectors.in.txt
+	    $(top_srcdir)/make-dissectors.prebuilt $@ @dissectors.in.txt
 
 MOSTLYCLEANFILES = \
 	dissectors.in.txt
