diff -uNr riscvemu-2017-08-06/fs.h riscvemu-2017-08-06.mod/fs.h
--- riscvemu-2017-08-06/fs.h	2017-08-06 15:30:12.000000000 +0300
+++ riscvemu-2017-08-06.mod/fs.h	2018-06-27 23:04:53.832086899 +0300
@@ -109,6 +109,16 @@
     uint64_t path;
 } FSQID;
 
+#ifdef st_atime_nsec
+#undef st_atime_nsec
+#endif
+#ifdef st_mtime_nsec
+#undef st_mtime_nsec
+#endif
+#ifdef st_ctime_nsec
+#undef st_ctime_nsec
+#endif
+
 typedef struct {
     FSQID qid;
     uint32_t st_mode;
