diff -uNr riscvemu-2017-08-06/riscv_machine.c riscvemu-2017-08-06.mod/riscv_machine.c
--- riscvemu-2017-08-06/riscv_machine.c	2017-08-06 15:30:12.000000000 +0300
+++ riscvemu-2017-08-06.mod/riscv_machine.c	2018-06-27 23:04:53.845420234 +0300
@@ -719,7 +719,7 @@
 #if 0
     {
         FILE *f;
-        f = fopen("/tmp/riscvemu.dtb", "wb");
+        f = fopen("@TERMUX_PREFIX@/tmp/riscvemu.dtb", "wb");
         fwrite(dst, 1, size, f);
         fclose(f);
     }
