diff -uNr qemu-2.10.1/linux-user/syscall.c qemu-2.10.1.mod/linux-user/syscall.c
--- qemu-2.10.1/linux-user/syscall.c	2017-10-03 00:23:41.000000000 +0300
+++ qemu-2.10.1.mod/linux-user/syscall.c	2017-11-29 23:56:05.096992811 +0200
@@ -7597,7 +7597,7 @@
         /* create temporary file to map stat to */
         tmpdir = getenv("TMPDIR");
         if (!tmpdir)
-            tmpdir = "/tmp";
+            tmpdir = "/data/data/com.termux/files/usr/tmp";
         snprintf(filename, sizeof(filename), "%s/qemu-open.XXXXXX", tmpdir);
         fd = mkstemp(filename);
         if (fd < 0) {
