diff -uNr qemu-2.11.1/migration/exec.c qemu-2.11.1.mod/migration/exec.c
--- qemu-2.11.1/migration/exec.c	2018-02-14 23:53:23.000000000 +0200
+++ qemu-2.11.1.mod/migration/exec.c	2018-06-24 01:50:55.109128121 +0300
@@ -28,7 +28,7 @@
 void exec_start_outgoing_migration(MigrationState *s, const char *command, Error **errp)
 {
     QIOChannel *ioc;
-    const char *argv[] = { "/bin/sh", "-c", command, NULL };
+    const char *argv[] = { "/data/data/com.termux/files/usr/bin/sh", "-c", command, NULL };
 
     trace_migration_exec_outgoing(command);
     ioc = QIO_CHANNEL(qio_channel_command_new_spawn(argv,
@@ -55,7 +55,7 @@
 void exec_start_incoming_migration(const char *command, Error **errp)
 {
     QIOChannel *ioc;
-    const char *argv[] = { "/bin/sh", "-c", command, NULL };
+    const char *argv[] = { "/data/data/com.termux/files/usr/bin/sh", "-c", command, NULL };
 
     trace_migration_exec_incoming(command);
     ioc = QIO_CHANNEL(qio_channel_command_new_spawn(argv,
