diff -uNr qemu-2.11.1/net/tap.c qemu-2.11.1.mod/net/tap.c
--- qemu-2.11.1/net/tap.c	2018-02-14 23:53:23.000000000 +0200
+++ qemu-2.11.1.mod/net/tap.c	2018-06-20 13:16:17.140476542 +0300
@@ -525,7 +525,7 @@
             *parg++ = helper_cmd;
             *parg++ = NULL;
 
-            execv("/bin/sh", args);
+            execv("/data/data/com.termux/files/usr/bin/sh", args);
         } else {
             /* assume helper is just the executable path name */
 
