diff -uNr qemu-2.10.1/block.c qemu-2.10.1.mod/block.c
--- qemu-2.10.1/block.c	2017-10-03 00:23:41.000000000 +0300
+++ qemu-2.10.1.mod/block.c	2017-11-29 23:55:42.186992827 +0200
@@ -538,7 +538,7 @@
     const char *tmpdir;
     tmpdir = getenv("TMPDIR");
     if (!tmpdir) {
-        tmpdir = "/var/tmp";
+        tmpdir = "/data/data/com.termux/files/usr/var/tmp";
     }
     if (snprintf(filename, size, "%s/vl.XXXXXX", tmpdir) >= size) {
         return -EOVERFLOW;
