diff -uNr qemu-2.11.1/slirp/slirp.c qemu-2.11.1.mod/slirp/slirp.c
--- qemu-2.11.1/slirp/slirp.c	2018-02-14 23:53:23.000000000 +0200
+++ qemu-2.11.1.mod/slirp/slirp.c	2018-06-20 13:16:17.163809877 +0300
@@ -133,7 +133,7 @@
         return 0;
     }
     old_stat = *cached_stat;
-    if (stat("/etc/resolv.conf", cached_stat) != 0) {
+    if (stat("/data/data/com.termux/files/usr/etc/resolv.conf", cached_stat) != 0) {
         return -1;
     }
     if (cached_stat->st_dev == old_stat.st_dev
@@ -157,7 +157,7 @@
     void *tmp_addr = alloca(addrlen);
     unsigned if_index;
 
-    f = fopen("/etc/resolv.conf", "r");
+    f = fopen("/data/data/com.termux/files/usr/etc/resolv.conf", "r");
     if (!f)
         return -1;
 
