diff -uNr ./android-ndk-aide/build/tools/build-ccache.sh ./android-ndk-aide.mod/build/tools/build-ccache.sh
--- ./android-ndk-aide/build/tools/build-ccache.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-ccache.sh	2018-06-18 00:58:37.438962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-cxx-stl.sh ./android-ndk-aide.mod/build/tools/build-cxx-stl.sh
--- ./android-ndk-aide/build/tools/build-cxx-stl.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-cxx-stl.sh	2018-06-18 00:58:37.595629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2013 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-docs.sh ./android-ndk-aide.mod/build/tools/build-docs.sh
--- ./android-ndk-aide/build/tools/build-docs.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-docs.sh	2018-06-18 00:58:37.518962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2013 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-gcc.sh ./android-ndk-aide.mod/build/tools/build-gcc.sh
--- ./android-ndk-aide/build/tools/build-gcc.sh	2015-08-04 13:24:03.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-gcc.sh	2018-06-18 00:58:37.485629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-gdbserver.sh ./android-ndk-aide.mod/build/tools/build-gdbserver.sh
--- ./android-ndk-aide/build/tools/build-gdbserver.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-gdbserver.sh	2018-06-18 00:58:37.432295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-gdb-stub.sh ./android-ndk-aide.mod/build/tools/build-gdb-stub.sh
--- ./android-ndk-aide/build/tools/build-gdb-stub.sh	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-gdb-stub.sh	2018-06-18 00:58:37.448962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 
 NDK_BUILDTOOLS_PATH="$(dirname $0)"
 . "$NDK_BUILDTOOLS_PATH/prebuilt-common.sh"
diff -uNr ./android-ndk-aide/build/tools/build-gnu-libstdc++.sh ./android-ndk-aide.mod/build/tools/build-gnu-libstdc++.sh
--- ./android-ndk-aide/build/tools/build-gnu-libstdc++.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-gnu-libstdc++.sh	2018-06-18 00:58:37.512295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-awk.sh ./android-ndk-aide.mod/build/tools/build-host-awk.sh
--- ./android-ndk-aide/build/tools/build-host-awk.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-awk.sh	2018-06-18 00:58:37.502295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-gcc.sh ./android-ndk-aide.mod/build/tools/build-host-gcc.sh
--- ./android-ndk-aide/build/tools/build-host-gcc.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-gcc.sh	2018-06-18 00:58:37.392295715 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2012 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-gdb.sh ./android-ndk-aide.mod/build/tools/build-host-gdb.sh
--- ./android-ndk-aide/build/tools/build-host-gdb.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-gdb.sh	2018-06-18 00:58:37.415629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2012 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-make.sh ./android-ndk-aide.mod/build/tools/build-host-make.sh
--- ./android-ndk-aide/build/tools/build-host-make.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-make.sh	2018-06-18 00:58:37.562295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-perl.sh ./android-ndk-aide.mod/build/tools/build-host-perl.sh
--- ./android-ndk-aide/build/tools/build-host-perl.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-perl.sh	2018-06-18 00:58:37.442295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2013 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-prebuilts.sh ./android-ndk-aide.mod/build/tools/build-host-prebuilts.sh
--- ./android-ndk-aide/build/tools/build-host-prebuilts.sh	2015-08-04 13:24:48.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-prebuilts.sh	2018-06-18 00:58:37.602295717 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-python.sh ./android-ndk-aide.mod/build/tools/build-host-python.sh
--- ./android-ndk-aide/build/tools/build-host-python.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-python.sh	2018-06-18 00:58:37.422295715 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2012 The Android Open Source Project
 # Copyright (C) 2012 Ray Donnelly <mingw.android at gmail.com>
diff -uNr ./android-ndk-aide/build/tools/build-host-toolbox.sh ./android-ndk-aide.mod/build/tools/build-host-toolbox.sh
--- ./android-ndk-aide/build/tools/build-host-toolbox.sh	2014-08-27 16:44:20.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-toolbox.sh	2018-06-18 00:58:37.525629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-host-yasm.sh ./android-ndk-aide.mod/build/tools/build-host-yasm.sh
--- ./android-ndk-aide/build/tools/build-host-yasm.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-host-yasm.sh	2018-06-18 00:58:37.555629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2013 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-llvm.py ./android-ndk-aide.mod/build/tools/build-llvm.py
--- ./android-ndk-aide/build/tools/build-llvm.py	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-llvm.py	2018-06-18 00:58:37.505629049 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright (C) 2015 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-mingw64-toolchain.sh ./android-ndk-aide.mod/build/tools/build-mingw64-toolchain.sh
--- ./android-ndk-aide/build/tools/build-mingw64-toolchain.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-mingw64-toolchain.sh	2018-06-18 00:58:37.548962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2012 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-ndk-stack.sh ./android-ndk-aide.mod/build/tools/build-ndk-stack.sh
--- ./android-ndk-aide/build/tools/build-ndk-stack.sh	2015-08-03 18:21:39.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-ndk-stack.sh	2018-06-18 00:58:37.618962384 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-ndk-sysroot.sh ./android-ndk-aide.mod/build/tools/build-ndk-sysroot.sh
--- ./android-ndk-aide/build/tools/build-ndk-sysroot.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-ndk-sysroot.sh	2018-06-18 00:58:37.568962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2009 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-support.sh ./android-ndk-aide.mod/build/tools/build-support.sh
--- ./android-ndk-aide/build/tools/build-support.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-support.sh	2018-06-18 00:58:37.535629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2013 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/build-target-prebuilts.sh ./android-ndk-aide.mod/build/tools/build-target-prebuilts.sh
--- ./android-ndk-aide/build/tools/build-target-prebuilts.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/build-target-prebuilts.sh	2018-06-18 00:58:37.608962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/check-glibc.sh ./android-ndk-aide.mod/build/tools/check-glibc.sh
--- ./android-ndk-aide/build/tools/check-glibc.sh	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/check-glibc.sh	2018-06-18 00:58:37.578962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 
 # Check if ELF depends on GLIBC/GLIBCXX symbols known to be absent from
 # libc.so in server
diff -uNr ./android-ndk-aide/build/tools/cleanup-apps.sh ./android-ndk-aide.mod/build/tools/cleanup-apps.sh
--- ./android-ndk-aide/build/tools/cleanup-apps.sh	2014-08-27 16:44:20.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/cleanup-apps.sh	2018-06-18 00:58:37.612295717 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # This is used to cleanup the project directories before making a commit or
 # a clean release. This will get rid of auto-generated files in the
diff -uNr ./android-ndk-aide/build/tools/dev-cleanup.sh ./android-ndk-aide.mod/build/tools/dev-cleanup.sh
--- ./android-ndk-aide/build/tools/dev-cleanup.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-cleanup.sh	2018-06-18 00:58:37.402295715 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/dev-platform-compress.sh ./android-ndk-aide.mod/build/tools/dev-platform-compress.sh
--- ./android-ndk-aide/build/tools/dev-platform-compress.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-platform-compress.sh	2018-06-18 00:58:37.542295716 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/dev-platform-expand-all.sh ./android-ndk-aide.mod/build/tools/dev-platform-expand-all.sh
--- ./android-ndk-aide/build/tools/dev-platform-expand-all.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-platform-expand-all.sh	2018-06-18 00:58:37.515629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/dev-platform-expand.sh ./android-ndk-aide.mod/build/tools/dev-platform-expand.sh
--- ./android-ndk-aide/build/tools/dev-platform-expand.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-platform-expand.sh	2018-06-18 00:58:37.575629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/dev-platform-import.sh ./android-ndk-aide.mod/build/tools/dev-platform-import.sh
--- ./android-ndk-aide/build/tools/dev-platform-import.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-platform-import.sh	2018-06-18 00:58:37.465629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/dev-system-import.sh ./android-ndk-aide.mod/build/tools/dev-system-import.sh
--- ./android-ndk-aide/build/tools/dev-system-import.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/dev-system-import.sh	2018-06-18 00:58:37.475629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/find-case-duplicates.sh ./android-ndk-aide.mod/build/tools/find-case-duplicates.sh
--- ./android-ndk-aide/build/tools/find-case-duplicates.sh	2014-08-27 16:44:20.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/find-case-duplicates.sh	2018-06-18 00:58:37.478962383 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2009 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/gen-platforms.sh ./android-ndk-aide.mod/build/tools/gen-platforms.sh
--- ./android-ndk-aide/build/tools/gen-platforms.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/gen-platforms.sh	2018-06-18 00:58:37.458962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/gen-system-symbols.sh ./android-ndk-aide.mod/build/tools/gen-system-symbols.sh
--- ./android-ndk-aide/build/tools/gen-system-symbols.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/gen-system-symbols.sh	2018-06-18 00:58:37.408962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2011 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/gen-toolchain-wrapper.sh ./android-ndk-aide.mod/build/tools/gen-toolchain-wrapper.sh
--- ./android-ndk-aide/build/tools/gen-toolchain-wrapper.sh	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/gen-toolchain-wrapper.sh	2018-06-18 00:58:37.425629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2012 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/ndk-ccache-gcc.sh ./android-ndk-aide.mod/build/tools/ndk-ccache-gcc.sh
--- ./android-ndk-aide/build/tools/ndk-ccache-gcc.sh	2014-08-27 16:44:20.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/ndk-ccache-gcc.sh	2018-06-18 00:58:37.528962383 +0300
@@ -1,2 +1,2 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 $NDK_CCACHE $NDK_CCACHE_CC "$@"
diff -uNr ./android-ndk-aide/build/tools/ndk-ccache-g++.sh ./android-ndk-aide.mod/build/tools/ndk-ccache-g++.sh
--- ./android-ndk-aide/build/tools/ndk-ccache-g++.sh	2014-08-27 16:44:20.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/ndk-ccache-g++.sh	2018-06-18 00:58:37.532295716 +0300
@@ -1,2 +1,2 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 $NDK_CCACHE $NDK_CCACHE_CXX "$@"
diff -uNr ./android-ndk-aide/build/tools/package-release.sh ./android-ndk-aide.mod/build/tools/package-release.sh
--- ./android-ndk-aide/build/tools/package-release.sh	2015-08-05 09:29:08.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/package-release.sh	2018-06-18 00:58:37.495629049 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2009-2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/patch-sources.sh ./android-ndk-aide.mod/build/tools/patch-sources.sh
--- ./android-ndk-aide/build/tools/patch-sources.sh	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/patch-sources.sh	2018-06-18 00:58:37.585629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/build/tools/rebuild-all-prebuilt.sh ./android-ndk-aide.mod/build/tools/rebuild-all-prebuilt.sh
--- ./android-ndk-aide/build/tools/rebuild-all-prebuilt.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/build/tools/rebuild-all-prebuilt.sh	2018-06-18 00:58:37.398962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/checkbuild.sh ./android-ndk-aide.mod/checkbuild.sh
--- ./android-ndk-aide/checkbuild.sh	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/checkbuild.sh	2018-06-18 00:58:37.232295714 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/data/data/com.termux/files/usr/bin/bash
 set -e
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
diff -uNr ./android-ndk-aide/ndk-build ./android-ndk-aide.mod/ndk-build
--- ./android-ndk-aide/ndk-build	2016-01-20 17:26:37.000000000 +0200
+++ ./android-ndk-aide.mod/ndk-build	2018-06-18 00:58:37.365629048 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/ndk-gdb ./android-ndk-aide.mod/ndk-gdb
--- ./android-ndk-aide/ndk-gdb	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/ndk-gdb	2018-06-18 00:58:37.645629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Copyright (C) 2010 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/ndk-gdb-py ./android-ndk-aide.mod/ndk-gdb-py
--- ./android-ndk-aide/ndk-gdb-py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/ndk-gdb-py	2018-06-18 00:58:37.378962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 
 PROGDIR=`dirname $0`
 PROGDIR=`cd $PROGDIR && pwd -P`
diff -uNr ./android-ndk-aide/ndk-gdb.py ./android-ndk-aide.mod/ndk-gdb.py
--- ./android-ndk-aide/ndk-gdb.py	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/ndk-gdb.py	2018-06-18 00:58:37.628962384 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 
 r'''
  Copyright (C) 2010 The Android Open Source Project
diff -uNr ./android-ndk-aide/ndk-which ./android-ndk-aide.mod/ndk-which
--- ./android-ndk-aide/ndk-which	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/ndk-which	2018-06-18 00:58:37.635629050 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/data/data/com.termux/files/usr/bin/bash
 #
 # Copyright (C) 2012 The Android Open Source Project
 #
diff -uNr ./android-ndk-aide/realpath ./android-ndk-aide.mod/realpath
--- ./android-ndk-aide/realpath	2015-07-30 15:12:05.000000000 +0300
+++ ./android-ndk-aide.mod/realpath	2018-06-18 00:58:37.372295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 """Provides the same functionality as realpath(1).
 
 Needed because Mac has no simple way of doing this from the command line.
diff -uNr ./android-ndk-aide/remove-windows-symlink.sh ./android-ndk-aide.mod/remove-windows-symlink.sh
--- ./android-ndk-aide/remove-windows-symlink.sh	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/remove-windows-symlink.sh	2018-06-18 00:58:37.358962382 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 
 . `dirname $0`/build/tools/ndk-common.sh
 
diff -uNr ./android-ndk-aide/sources/android/support/tests/run-host.sh ./android-ndk-aide.mod/sources/android/support/tests/run-host.sh
--- ./android-ndk-aide/sources/android/support/tests/run-host.sh	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/android/support/tests/run-host.sh	2018-06-18 00:58:37.265629048 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 
 set -e
 export LANG=C
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++/libcxx/test/testit ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/libcxx/test/testit
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++/libcxx/test/testit	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/libcxx/test/testit	2018-06-18 00:58:37.258962381 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 # //===--------------------------- testit ---------------------------------===//
 # //
 # //                     The LLVM Compiler Infrastructure
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++/libcxx/test/testit_android ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/libcxx/test/testit_android
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++/libcxx/test/testit_android	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/libcxx/test/testit_android	2018-06-18 00:58:37.255629048 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 # //===--------------------------- testit ---------------------------------===//
 # //
 # //                     The LLVM Compiler Infrastructure
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++/tools/upgrade-upstream.sh ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/tools/upgrade-upstream.sh
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++/tools/upgrade-upstream.sh	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++/tools/upgrade-upstream.sh	2018-06-18 00:58:37.248962381 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # A small script used to update the content of libcxx/ to a newer
 # version of libc++.
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/lib/buildit ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/lib/buildit
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/lib/buildit	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/lib/buildit	2018-06-18 00:58:37.248962381 +0300
@@ -1,4 +1,4 @@
-#! /bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 #
 # Set the $TRIPLE environment variable to your system's triple before
 # running this script.  If you set $CXX, that will be used to compile
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit	2015-07-29 15:22:28.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit	2018-06-18 00:58:37.235629048 +0300
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/data/data/com.termux/files/usr/bin/sh
 # //===--------------------------- testit ---------------------------------===//
 # //
 # //                     The LLVM Compiler Infrastructure
diff -uNr ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit_android ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit_android
--- ./android-ndk-aide/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit_android	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/cxx-stl/llvm-libc++abi/libcxxabi/test/testit_android	2018-06-18 00:58:37.242295714 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/data/data/com.termux/files/usr/bin/bash
 # //===--------------------------- testit ---------------------------------===//
 # //
 # //                     The LLVM Compiler Infrastructure
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_break_on_failure_unittest.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_break_on_failure_unittest.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_break_on_failure_unittest.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_break_on_failure_unittest.py	2018-06-18 00:58:37.312295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2006, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_catch_exceptions_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_catch_exceptions_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_catch_exceptions_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_catch_exceptions_test.py	2018-06-18 00:58:37.305629048 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2010 Google Inc.  All Rights Reserved.
 #
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_color_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_color_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_color_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_color_test.py	2018-06-18 00:58:37.348962382 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2008, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_env_var_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_env_var_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_env_var_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_env_var_test.py	2018-06-18 00:58:37.292295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2008, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_filter_unittest.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_filter_unittest.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_filter_unittest.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_filter_unittest.py	2018-06-18 00:58:37.285629048 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2005 Google Inc. All Rights Reserved.
 #
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_help_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_help_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_help_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_help_test.py	2018-06-18 00:58:37.342295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2009, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_list_tests_unittest.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_list_tests_unittest.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_list_tests_unittest.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_list_tests_unittest.py	2018-06-18 00:58:37.272295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2006, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_output_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_output_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_output_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_output_test.py	2018-06-18 00:58:37.352295715 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2008, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_shuffle_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_shuffle_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_shuffle_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_shuffle_test.py	2018-06-18 00:58:37.335629048 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2009 Google Inc. All Rights Reserved.
 #
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_test_utils.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_test_utils.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_test_utils.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_test_utils.py	2018-06-18 00:58:37.315629048 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2006, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_throw_on_failure_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_throw_on_failure_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_throw_on_failure_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_throw_on_failure_test.py	2018-06-18 00:58:37.328962382 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2009, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_uninitialized_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_uninitialized_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_uninitialized_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_uninitialized_test.py	2018-06-18 00:58:37.268962381 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2008, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_outfiles_test.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_outfiles_test.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_outfiles_test.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_outfiles_test.py	2018-06-18 00:58:37.298962381 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2008, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_output_unittest.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_output_unittest.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_output_unittest.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_output_unittest.py	2018-06-18 00:58:37.325629048 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2006, Google Inc.
 # All rights reserved.
diff -uNr ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_test_utils.py ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_test_utils.py
--- ./android-ndk-aide/sources/third_party/googletest/googletest/test/gtest_xml_test_utils.py	2015-07-29 15:05:35.000000000 +0300
+++ ./android-ndk-aide.mod/sources/third_party/googletest/googletest/test/gtest_xml_test_utils.py	2018-06-18 00:58:37.278962381 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/data/com.termux/files/usr/bin/env python
 #
 # Copyright 2006, Google Inc.
 # All rights reserved.
