diff -uNr dwm-6.1.mod/config.def.h dwm-6.1.mod2/config.def.h
--- dwm-6.1.mod/config.def.h	2015-11-09 00:39:37.000000000 +0200
+++ dwm-6.1.mod2/config.def.h	2018-04-11 12:15:31.046011185 +0300
@@ -50,7 +50,7 @@
 	{ MODKEY|ControlMask|ShiftMask, KEY,      toggletag,      {.ui = 1 << TAG} },
 
 /* helper for spawning shell commands in the pre dwm-5.0 fashion */
-#define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
+#define SHCMD(cmd) { .v = (const char*[]){ "@TERMUX_PREFIX@/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
