diff -uNr st-0.8.1.mod/config.def.h st-0.8.1.mod2/config.def.h
--- st-0.8.1.mod/config.def.h	2018-03-20 22:29:59.000000000 +0200
+++ st-0.8.1.mod2/config.def.h	2018-04-11 12:03:45.766720886 +0300
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVu Sans Mono:pixelsize=12:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -16,7 +16,7 @@
  * 4: value of shell in /etc/passwd
  * 5: value of shell in config.h
  */
-static char *shell = "/bin/sh";
+static char *shell = "@TERMUX_PREFIX@/bin/sh";
 char *utmp = NULL;
 char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";
 
