diff -uNr feh-2.25.1/src/imlib.c feh-2.25.1.mod/src/imlib.c
--- feh-2.25.1/src/imlib.c	2018-03-07 18:53:33.000000000 +0200
+++ feh-2.25.1.mod/src/imlib.c	2018-06-21 17:31:53.793526576 +0300
@@ -102,7 +102,7 @@
 {
 	/* Set up the font stuff */
 	imlib_add_path_to_font_path(".");
-	imlib_add_path_to_font_path(PREFIX "/share/feh/fonts");
+	imlib_add_path_to_font_path("/data/data/com.termux/files/usr/share/feh/fonts");
 
 	return;
 }
@@ -320,7 +320,7 @@
 	else
 		basename++;
 
-	tmpname = feh_unique_filename("/tmp/", basename);
+	tmpname = feh_unique_filename("/data/data/com.termux/files/usr/tmp/", basename);
 
 	if (strlen(tmpname) > (NAME_MAX-6))
 		tmpname[NAME_MAX-7] = '\0';
