diff -uNr minetest-0.4.17.1/src/main.cpp minetest-0.4.17.1.mod/src/main.cpp
--- minetest-0.4.17.1/src/main.cpp	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/main.cpp	2018-09-04 17:29:10.270811370 +0300
@@ -141,13 +141,7 @@
 	setup_log_params(cmd_args);
 
 	porting::signal_handler_init();
-
-#ifdef __ANDROID__
-	porting::initAndroid();
-	porting::initializePathsAndroid();
-#else
 	porting::initializePaths();
-#endif
 
 	if (!create_userdata_path()) {
 		errorstream << "Cannot create user data directory" << std::endl;
@@ -178,12 +172,10 @@
 	if (g_settings->getBool("enable_console"))
 		porting::attachOrCreateConsole();
 
-#ifndef __ANDROID__
 	// Run unit tests
 	if (cmd_args.getFlag("run-unittests")) {
 		return run_tests();
 	}
-#endif
 
 	GameParams game_params;
 #ifdef SERVER
@@ -412,17 +404,8 @@
 {
 	bool success;
 
-#ifdef __ANDROID__
-	if (!fs::PathExists(porting::path_user)) {
-		success = fs::CreateDir(porting::path_user);
-	} else {
-		success = true;
-	}
-	porting::copyAssets();
-#else
 	// Create user data directory
 	success = fs::CreateDir(porting::path_user);
-#endif
 
 	return success;
 }
