diff -uNr minetest-0.4.17.1/src/config.h minetest-0.4.17.1.mod/src/config.h
--- minetest-0.4.17.1/src/config.h	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/config.h	2018-09-04 17:29:10.267478019 +0300
@@ -12,16 +12,6 @@
 
 #if defined USE_CMAKE_CONFIG_H
 	#include "cmake_config.h"
-#elif defined (__ANDROID__) || defined (ANDROID)
-	#define PROJECT_NAME "minetest"
-	#define PROJECT_NAME_C "Minetest"
-	#define STATIC_SHAREDIR ""
-	#include "android_version.h"
-	#ifdef NDEBUG
-		#define BUILD_TYPE "Release"
-	#else
-		#define BUILD_TYPE "Debug"
-	#endif
 #else
 	#ifdef NDEBUG
 		#define BUILD_TYPE "Release"
