diff -uNr minetest-0.4.17.1/src/localplayer.cpp minetest-0.4.17.1.mod/src/localplayer.cpp
--- minetest-0.4.17.1/src/localplayer.cpp	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/localplayer.cpp	2018-09-04 15:42:26.140154371 +0300
@@ -341,11 +341,6 @@
 	// TODO: this shouldn't be hardcoded but transmitted from server
 	float player_stepheight = (touching_ground) ? (BS * 0.6f) : (BS * 0.2f);
 
-#ifdef __ANDROID__
-	if (touching_ground)
-		player_stepheight += (0.6f * BS);
-#endif
-
 	v3f accel_f = v3f(0,0,0);
 
 	collisionMoveResult result = collisionMoveSimple(env, m_client,
@@ -929,10 +924,6 @@
 	// this shouldn't be hardcoded but transmitted from server
 	float player_stepheight = touching_ground ? (BS * 0.6) : (BS * 0.2);
 
-#ifdef __ANDROID__
-	player_stepheight += (0.6 * BS);
-#endif
-
 	v3f accel_f = v3f(0, 0, 0);
 
 	collisionMoveResult result = collisionMoveSimple(env, m_client,
