diff -uNr minetest-0.4.17.1/src/drawscene.cpp minetest-0.4.17.1.mod/src/drawscene.cpp
--- minetest-0.4.17.1/src/drawscene.cpp	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/drawscene.cpp	2018-09-04 17:29:10.267478019 +0300
@@ -630,16 +630,9 @@
 		video::ITexture *progress_img_bg = tsrc->getTexture("progress_bar_bg.png");
 
 		if (progress_img && progress_img_bg) {
-#ifndef __ANDROID__
 			const core::dimension2d<u32> &img_size = progress_img_bg->getSize();
 			u32 imgW = rangelim(img_size.Width, 200, 600);
 			u32 imgH = rangelim(img_size.Height, 24, 72);
-#else
-			const core::dimension2d<u32> img_size(256, 48);
-			float imgRatio = (float) img_size.Height / img_size.Width;
-			u32 imgW = screensize.X / 2.2f;
-			u32 imgH = floor(imgW * imgRatio);
-#endif
 			v2s32 img_pos((screensize.X - imgW) / 2, (screensize.Y - imgH) / 2);
 
 			draw2DImageFilterScaled(
