diff -uNr minetest-0.4.17.1/src/guiFormSpecMenu.h minetest-0.4.17.1.mod/src/guiFormSpecMenu.h
--- minetest-0.4.17.1/src/guiFormSpecMenu.h	2018-06-10 18:18:11.000000000 +0300
+++ minetest-0.4.17.1.mod/src/guiFormSpecMenu.h	2018-09-04 17:29:10.270811370 +0300
@@ -365,10 +365,6 @@
 	GUITable* getTable(const std::string &tablename);
 	std::vector<std::string>* getDropDownValues(const std::string &name);
 
-#ifdef __ANDROID__
-	bool getAndroidUIInput();
-#endif
-
 protected:
 	v2s32 getBasePos() const
 	{
@@ -539,10 +535,6 @@
 
 	std::wstring getLabelByID(s32 id);
 	std::string getNameByID(s32 id);
-#ifdef __ANDROID__
-	v2s32 m_down_pos;
-	std::string m_JavaDialogFieldName;
-#endif
 
 	/* If true, remap a double-click (or double-tap) action to ESC. This is so
 	 * that, for example, Android users can double-tap to close a formspec.
